messages/whitespace-found.txt

```diff
@@ -0,0 +1,3 @@
+ESLint couldn't find the plugin "eslint-plugin-<%- pluginName %>". It looks like there is whitespace in the plugin name.
```

Comment:
Yes, `longName` includes `eslint-plugin-` part.
[This commit](https://github.com/eslint/eslint/pull/6939/files#diff-2cf43403ee6d1d790832b4dfa8905d73L1) also removed the part.